.github/workflows/issue-auto-assign.yml

```diff
@@ -0,0 +1,24 @@
+name: 'Auto assign maintainer to issue'
+on:
+  issues:
+    types: [opened]
+
+permissions:
+  issues: write
+
+jobs:
+  assign-user:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v6
+        with:
+          script: |
+            const assignees = ['mstoykov', 'imiric, 'codebien', 'olegbespalov', 'oleiade'];
+            const assigneeCount = 1;
+
+            github.rest.issues.addAssignees({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              issue_number: context.issue.number,
+              assignees: assignees.sort(() => 0.5 - Math.random()).slice(0, assigneeCount),
```

Comment:
Should this ignore issues opened by at least the `assignees`?

Reply:
or maybe just directly assigning it to the assignee that opened it :thinking:

Reply:
See https://github.com/grafana/k6/pull/3045/commits/d93974d9232fc8f79df1d19be2c162b7f2fb9676